cg/meta/workflow/nf_analysis.py

```diff
@@ -629,6 +629,7 @@ def get_formatted_file_deliverable(
                 .replace("SAMPLENAME", sample_name)
                 .replace("PATHTOCASE", case_path)
             )
+            LOG.info(deliverables[deliverable_field])
```

Comment:
This looks more relevant for developers rather than production
```suggestion
            LOG.debug(deliverables[deliverable_field])
```